andru1989/vowels.rb

```diff
@@ -37,26 +40,57 @@ def read_file
     print_vowels
   end
 
+  # Public
+  #
+  #   => Set initial accessor with Time.now
+  #   => Get the content of 'text.txt' file and loop each slice with(20000)
+  #   => Acumulate in '@threads' Array a new Thread
+  #   => For each ary it call the 'set_values' method to update de Hash
+  #   => Wait a moment while each Thread finish
+  #   => Set final accessor with Time.now
+  #   => Then it call the 'print_vowels' method to print the hash
+  def read_thread_file
+    self.initial = Time.now
+    open_file.each_slice(20000) do |ary|
+      @threads << Thread.new do
+        set_values(ary)
+      end
+    end
+    @threads.each{|t| t.join}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:join` as an argument to `each` instead of a block.